pycbc/vetoes/sgchisq.py

```diff
@@ -121,12 +121,23 @@ def values(self, stilde, template, psd, snrv, snr_norm,
             # Shift the time of interest to be centered on 0
             stilde_shift = apply_fseries_time_shift(stilde, -time)
 
+            # Only apply the sine-Gaussian in a +-50 Hz range around the 
+            # central frequency
             qwindow = 50
             chisq[i] = 0
-            #Find fpeak based on difference from beginning of last chisq
-            #point
-            fpeak = (bins[-2] * 2.0 - bins[-3]) * template.delta_f
-            fstop = len(template) * template.delta_f * 0.9
+            # Determine the final frequency of the waveform. We don't know this
+            # exactly so we make use of the chisq bins to estimate.
+            # The edge with a meaningful number is the second to last, so we
```

Comment:
i guess it's not clear to me if the last bin is expected to contain any power : is it that the last bin covers the last 1/p of the signal power plus all the frequency over which the waveform just vanishes?